src/models/common/PlexMediaTypeString.yaml

```
@@ -12,3 +15,6 @@ x-speakeasy-enums:
   - TV_SHOW
   - SEASON
   - EPISODE
+  - AUDIO
```

```suggestion
  - ARTIST
```

@angafirith Should this not be `ARTIST`?

---

Apologies for the late response.

I believe AUDIO is correct because that's the value used in [src/models/common/PlexMediaType.yaml](src/models/common/PlexMediaType.yaml). I think the primary thing here is that these values need to be in alignment with each other. (i.e. the string value in the response needs to map to the same x-speakeasy-enum value as the int value in the request).

I have to admit that I'm not entirely sure here, since I haven't had the chance to figure out how to regenerate the go module using the updated API spec. I figured out these values by manually editing the source of that module, but there could be some nuance here that I am missing.

---

Looking at it again, it should be `ARTIST`, since this will be the enum value that corresponds with `artist` as a string value. The order here matters, as with the other options. If you change it, I will merge it